website/static/js/contribAdder.js

```diff
@@ -446,6 +448,7 @@ AddContributorViewModel = oop.extend(Paginator, {
         }).fail(function (xhr, status, error) {
             self.hide();
             $osf.unblock();
+            self.canSubmit(true);
```

Comment:
Can we put this and line 438-441 and 449-451 in a `complete` or an `always` method and consolidate these?